bmc/templates/bmc_history.html

```diff
@@ -0,0 +1,19 @@
+{% extends "layout.html" %}
+
+{% block app_content %}
+
+<h1>
+    Previous tries for {{ user.username }}:
+</h1>
+<hr>
+{% for practice in practices %}
+<p>{{ practice.timestamp }}: {{ practice.correct }} correct in Book {{ practice.Book }} using {{ practice.medium }}</p>
+{% endfor %}
```

Comment:
Instead of `{% endfor %}` maybe use an `{% else %}` in between (see https://jinja.palletsprojects.com/en/2.11.x/templates/#for) in case there were no practices